app/notifications/process_notifications.py

```diff
@@ -32,11 +32,10 @@
 )
 
 from app.v2.errors import BadRequestError
-from app.utils import get_template_instance
```

Comment:
Can we delete `get_template_instance` now?

Reply:
It’s still used in a few places where we only have template JSON, not a database object.